docs/platforms/apple/common/user-feedback/index.mdx

```diff
@@ -1,15 +1,113 @@
 ---
 title: Set Up User Feedback
-description: "Learn more about collecting user feedback when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
+description: "Learn how to enable User Feedback in your Cocoa app."
 sidebar_order: 6000
 ---
 
-When a user experiences an error, Sentry provides the ability to collect additional feedback. You can collect feedback according to the method supported by the SDK.
+The User Feedback feature allows you to collect user feedback from anywhere inside your application at any time, without needing an error event to occur first.
 
-## User Feedback API
+Note that if you're using a self-hosted Sentry instance, you'll need to be on version 24.4.2 or higher in order to use the full functionality of the User Feedback feature. Lower versions may have limited functionality.
 
-The user feedback API allows you to collect user feedback while utilizing your own UI. You can use the same programming language you have in your app to send user feedback. In this case, the SDK creates the HTTP request so you don't have to deal with posting data via HTTP.
+## User Feedback Widget
 
-Sentry pairs the feedback with the original event, giving you additional insight into issues. Sentry needs the `eventId` to be able to associate the user feedback to the corresponding event. For example, to get the `eventId`, you can use <PlatformLink to="/configuration/options/#before-send"><PlatformIdentifier name="before-send" /></PlatformLink> or the return value of the method capturing an event.
+The User Feedback widget allows users to submit feedback from anywhere inside your application.
 
-<PlatformContent includePath="user-feedback/sdk-api-example/" />
+### Pre-requisites
+
+Ensure that your project is set up with Sentry and that you have added the Sentry Cocoa SDK to your project.
+
+### Installation
+
+1. Install the Sentry Cocoa SDK using CocoaPods, Carthage, or Swift Package Manager.
+2. Ensure you are using version 8.35.0 or above of the SDK to access the latest features.
+
+### Set Up
+
+To start using the User Feedback widget in your Cocoa application, enable the feedback integration and provide an optional configuration in your SentryOptions when starting the SDK:
+
+```swift
+ SentrySDK.start { options in
+     options.enableUserFeedbackIntegration = true
+     options.configureUserFeedback = { config in
```

Comment:
Are these two related to teh widget only? Since we have `captureFeedback` I find confusing to have 'enableFeedback' and 'configureFeedback' options

Reply:
let's keep discussion of the nomenclature in https://github.com/getsentry/sentry-cocoa/pull/4284